pkg/cbfs/image.go

```diff
@@ -133,14 +128,20 @@ func (i *Image) Update() error {
 		if err := s.Write(&b); err != nil {
 			return err
 		}
-		Debug("Copy %d bytes to i.Data[%d]", len(b.Bytes()), s.Header().RecordStart+512)
-		copy(i.Data[s.Header().RecordStart+512:], b.Bytes())
+		end := uint32(len(b.Bytes())) + s.Header().RecordStart
+		if end > i.Area.Size {
+			return fmt.Errorf("write outside of CBFS not supported %x %x", end, i.Area.Size)
```

Comment:
it won't be clear to me what the numbers mean and hence I'll be confused by this message. 
Could you say something like
"Region [%#x, %#x] is outside [%#x,#x]" Do regions have names? From this error I won't be able to tell which region it is.